src/cloud-api-adaptor/podvm-mkosi/Makefile

```diff
@@ -60,79 +68,60 @@ endif
 		--build-arg VERIFY_PROVENANCE=$(VERIFY_PROVENANCE) \
 		$(if $(AUTHFILE),--build-arg AUTHFILE=$(AUTHFILE),) \
 		$(if $(DEFAULT_AGENT_POLICY_FILE),--build-arg DEFAULT_AGENT_POLICY_FILE=$(DEFAULT_AGENT_POLICY_FILE),) \
-		-o type=local,dest="./resources/binaries-tree" \
+		$(if $(filter $(PUSH),true),,-o type=local,dest="./resources/binaries-tree") \
+		$(DOCKER_OPTS) \
 		-f ../podvm/Dockerfile.podvm_binaries.fedora ../../
 
 PHONY: image
 image:
 	@echo "Enabling production preset..."
 	rm -rf resources/build*Image
-	rm -rf ./build
+	sudo rm -rf ./build
```

Comment:
can we avoid sudo here? it would be annoying for local or in-docker runs. we can maybe call make with sudo in the workflow.

Reply:
Yeah, I'll try and revert this commit and call sudo from the workflow. Thanks for the idea!